pkg/custmsg/custom_message.go

```diff
@@ -49,6 +49,38 @@ func (c Labeler) SendLogAsCustomMessage(msg string) error {
 	return sendLogAsCustomMessageW(msg, c.labels)
 }
 
+type MetricsLabeler struct {
```

Comment:
I think this does make sense in common, thanks for moving it over. Can it be in its own package, separate from `custmsg`?

Reply:
Sure thing

Reply:
Moved to `monitoring` after our convo.